.vscode/settings.json

```diff
@@ -6,5 +6,7 @@
   "[typescript]": {
     "editor.formatOnSave": false
   },
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "editor.wordWrapColumn": 150,
```

Comment:
I am confused.  I asked you to remove line 10, but you added line 11, which just makes it worse.

Reply:
Hmm the request said undo change to this line, that confused me